Makefile

```diff
@@ -4,7 +4,7 @@ REPODIR=/tmp/tf-repo/providers
 
 NAME=shoreline
 BINARY=terraform-provider-$(NAME)
-VERSION=1.15.22
+VERSION=1.15.23
```

Comment:
I believe you already have a PR up that is bumped to the same .23 minor version. Depending on the merge order, these should be incremented separately

Reply:
Yes, I have two others. I will update all accordingly once I start to merge them